### PR TITLE
[docs] Add 7.6 breaking changes and release highlights

### PR DESCRIPTION
**Pull request by dedemorton**
Please respond to questions that appear as comments within the source files.

### CODE REVIEW

File: libbeat/docs/release-notes/highlights/highlights-7.6.0.asciidoc [hunk 1]
```diff
@@ -18,8 +18,58 @@ For a list of bug fixes and other changes, see the {beats}
...
-//[float]
-//==== Highlight
+[float]
+==== Support added for TLS 1.3
+
+In 7.6, we've added support for TLS 1.3, enabling you to take advantage
+of the security and performance improvements available in TLS 1.3 when {beats}
+connects to an {es} cluster that supports it.
+
+//REVIEWERS: Any more details to add here. GitHub issues/PRs were spare.
```
Comment (urso) [inline libbeat/docs/release-notes/highlights/highlights-7.6.0.asciidoc]: That's it basically. Default enabled versions will be 1.1, 1.2, and 1.3. Older Beats used to enable 1.0, 1.1, 1.2 by default.

In general TLS 1.0 and 1.1 are not deemed secure enough anymore.

File: libbeat/docs/release-notes/highlights/highlights-7.6.0.asciidoc [hunk 2]
```diff
@@ -18,8 +18,58 @@ For a list of bug fixes and other changes, see the {beats}
...
+Starting in 7.6, you can use {functionbeat} to get events from 
+https://cloud.google.com/pubsub/[Google Pub/Sub] and
+https://cloud.google.com/storage/[Google Cloud Storage].
+
+You define functions that respond to Google triggers then export them as YAML
+files that you deploy by using the Google Cloud Deployment Manager. 
+
+// REVIEWERS: I can't say much about this yet because I just started working on
+// the docs. Please let me know what details we should include here and how we
+// want to position this.
```
Comment (urso) [inline libbeat/docs/release-notes/highlights/highlights-7.6.0.asciidoc]: @kvch Can you provide some details please? 

Some other GCP features/improvements that made it into 7.6:
- https://github.com/elastic/beats/pull/14829
- https://github.com/elastic/beats/pull/15200
- https://github.com/elastic/beats/pull/14553

Comment (dedemorton) [reply libbeat/docs/release-notes/highlights/highlights-7.6.0.asciidoc]: awesome! Thanks for the links. On Monday, I'll beef up the google cloud section to include the other improvements!

Comment (dedemorton) [reply libbeat/docs/release-notes/highlights/highlights-7.6.0.asciidoc]: @kvch I see there's a little blurb in the Logging blog, so I'll just use that. Thanks!

**Pull request merged**